doc/src/installation.md

```diff
@@ -1,319 +1,199 @@
 # Installation
 
-All the instructions below work on Linux, macOS and Windows.
+Follow the steps below to install and run LPython on Linux, Windows or macOS.
+
+## Prerequisites
+- ### Install Conda
+  Follow the instructions provided [here](https://github.com/conda-forge/miniforge/#download) to install Conda on your platform (Linux, macOS and Windows) using a conda-forge distribution called Miniforge.
+  
+  For Windows, these are additional requirements:
+  - Miniforge Prompt
+  - Visual Studio (with "Desktop Development with C++" workload)
+
+- ### Set up your system
+    - Linux
+        - Make sure you have `g++` and `cmake` installed. If not, install them using the below command:
+
+            ```bash
+            sudo apt-get install g++ cmake
```

Comment:
Since we are installing `build-essentials` below, I think g++ would not be needed to be explicitly installed.

Reply:
You are right. I missed this as I had installed `g++` separately before building the project, getting the error, and installing `build-essential`.